dora/core/common/src/main/java/alluxio/RuntimeConstants.java

```diff
@@ -30,11 +30,10 @@ public final class RuntimeConstants {
       ALLUXIO_DOCS_URL = "https://docs.alluxio.io/os/user/edge";
       ALLUXIO_JAVADOC_URL = "https://docs.alluxio.io/os/javadoc/edge";
     } else {
```

Comment:
need to remove `!VERSION.contains("\\.")` from the if condition to reach this part